guides/appendix/live-reload.md

```diff
@@ -68,7 +55,7 @@ interacting with the tailwind CLI a little easier.
 ```diff {data-filename="package.json"}
 + "tailwind:build": "npx tailwindcss -i ./tailwind-input.css -o ./public/assets/tailwind.css",
 + "tailwind:watch": "npx tailwindcss -i ./tailwind-input.css -o ./public/assets/tailwind.css --watch",
-+ "build": "npm run tailwind:build && ember build --environment=production",
++ "build": "npm run tailwind:build -- --minify && ember build --environment=production",
```

Comment:
I don't use `npm` myself, so I don't know if ` -- ` after `run` is still needed nowadays?

Reply:
Just tested, seems it's still needed.

Reply:
Nevermind, should've added this to `tailwind:build` in the first place 😬.